MicrowaveGame2D/Assets/Scripts/Events/EventId.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Events
 {
-    public readonly struct EventId
+    public readonly struct EventId<T>
```

Comment:
T should have the same constraint used in EventManager (T where T : EventArgs)

Reply:
Resolved in latest commit.